### ISSUE TITLE
Toolbar option to switch to all photos in the gallery 

### ISSUE THREAD

**Issue by pa1pal** [fossasia/phimpme-android#435]
an option menu to switch to all photos from albums mode in gallery.

**Comment by vinaysajjanapu** [comment]
@pa1pal assign me this

**Comment by vinaysajjanapu** [comment]
@pa1pal I'll work on it

**Comment by vinaysajjanapu** [comment]
After #434.

**Comment by anantprsd5** [comment]
I'll work on this.

**Comment by vinaysajjanapu** [comment]
@anantprsd5 #447 might help. Its implementation is available there.

_Sent from my Xiaomi Redmi Note 4X using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by anantprsd5** [comment]
@vinaysajjanapu It seems you have already done this in the PR #447. So I think it will be better if you are willing to take this issue? or else I can do.

**Comment by vinaysajjanapu** [comment]
@anantprsd5 Ok sure I am willing to work on this. I am currently working on image filters. It will be done by tomorrow. I will work on this after that. As I already implemented this, it won't take much time. 

_Sent from my Xiaomi Redmi Note 4X using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_
